server/api/auth/login.ts

```diff
@@ -19,7 +19,7 @@ const authenticationSuccessful = (req: Request, user: User, next: NextFunction,
       // More here:
       // https://github.com/voxpelli/node-connect-pg-simple/issues/31#issuecomment-230596077
       req.session.save(() => {
-        done(`${appUri}`)
+        done(`${process.env.NODE_ENV === 'development' ? '/' : appUri}`)
```

Comment:
why this?

Reply:
Fixes issue redirecting after login in development (no more error after login :) )